pkg/qemu/qemu.go

```diff
@@ -284,6 +284,10 @@ func getMacOSProductVersion() (*semver.Version, error) {
 		return nil, fmt.Errorf("failed to execute %v: %w", cmd.Args, err)
 	}
 	verTrimmed := strings.TrimSpace(string(b))
+	// macOS 12.4 returns just "12.4\n"
+	for strings.Count(verTrimmed, ".") < 2 {
+		verTrimmed += ".0"
```

Comment:
+= ".0.0" when count == 0

Reply:
That's why it is in a `for` loop, so it will loop twice in that case: https://go.dev/play/p/j_0P1mZWRtj